trillian/ctfe/services.go

```diff
@@ -37,8 +46,17 @@ func newIssuanceChainService(s storage.IssuanceChainStorage, c cache.IssuanceCha
 	return service
 }
 
+func (s *issuanceChainService) IsCTFEStorageEnabled() bool {
```

Comment:
It looks like this method is only used within this class, so it can be unexported?

You haven't taken my advice to split this service into two concrete implementations and I can live with that, but I think it's a missed opportunity to apply good OO design principles to achieve separation of concerns. The more methods on a class switch their behaviour based on the same condition (in this case `if !isCTFEStorageEnabled()`), the stronger the signal that this is better modelled as two different implementations.

Reply:
Unexported to `isCTFEStorageEnabled()`.

Your advice is 100% valid. I will come back to this (in another pull request) when time permits.